libraries/stdlib/jvm/src/kotlin/text/StringsJVM.kt

```diff
@@ -424,8 +424,14 @@ public inline fun String.toPattern(flags: Int = 0): java.util.regex.Pattern {
  *
  * @sample samples.text.Strings.capitalize
  */
-public actual fun String.capitalize(): String {
-    return if (isNotEmpty() && this[0].isLowerCase()) substring(0, 1).toUpperCase() + substring(1) else this
+public actual fun String.capitalize() = capitalize(Locale.getDefault())
```

Comment:
Better to leave the previous implementation to avoid accidentally changing behavior in 1.3.x release.